doc-content/enterprise-only/openshift/rhpam71-prod-immutable-monitor-ref.adoc

```diff
@@ -0,0 +1,333 @@
+////
+    AUTOGENERATED FILE - this file was generated via ./tools/gen_template_docs.py.
+    Changes to .adoc or HTML files may be overwritten! Please change the
+    generator or the input template (./*.in)
+////
+= rhpam71-prod-immutable-monitor
+[id='rhpam71-prod-immutable-monitor-ref']
+
+Application template for a router and monitoring console in a production environment, for Red Hat Process Automation Manager 7.1
+
+toc::[]
+
+
+== Parameters
+
+Templates allow you to define parameters which take on a value. That value is then substituted wherever the parameter is referenced.
```

Comment:
Tiny, but technically "that take on". Comes up several times.

Reply:
Sadly that's the autogenerated part. Might have to stay for now as I don't control the script.